modules/ShipStation/Api/docs/version_info/5.5/0.1/changelog.txt

```diff
@@ -0,0 +1,2 @@
+Version 5.5.0.0
```

Comment:
`Version: 5.5.0.1`? 

Also, why `5.5.0.0` and `5.5.0.1`? Is there a difference?

Reply:
This should still say `Version 5.5.0.1`